src/hotspot/share/gc/serial/genMarkSweep.cpp

```diff
@@ -182,7 +182,7 @@ void GenMarkSweep::mark_sweep_phase1(bool clear_all_softrefs) {
   ClassLoaderDataGraph::clear_claimed_marks();
 
   {
-    StrongRootsScope srs(1);
+    StrongRootsScope srs(0);
 
     gch->full_process_roots(&srs,
                             false, // not the adjust phase
```

Comment:
The `srs` parameter is also only used to get the number of threads in the callees.
It might be better to remove this parameter similar to the other comment in `serialHeap.cpp` as all that and related code is only used in Serial GC and all the parallel paths here can be removed since CMS is no more.

Reply:
I only removed its usage in the body, but forgot it in the argument list. Good catch.